### DISCUSSION TITLE
关于公式输出可能不是latex形式的问题

### DISCUSSION THREAD

**Discussion by JSYUs** [binary-husky/gpt_academic#312] [Ideas]
感谢作者的开发, 使用了一段时间, 今天体验了最新版, 感觉甚为良好!!!点赞

有个小问题, 当API返回的公式可能不是latex的形式, 那么最终输出结果就是乱糟糟, 不便于阅读的公式, 如返回的是 y_i = a + x_i, 但是没有$$符号, 而最终输出结果仍是 y_i = a + x_i. 

例如在input框内"请你帮我写一个时间序列公式", 他可能会直接输出不太美观的数学公式形式.
但是如果是"请你帮我写一个时间序列公式, 并使用latex输出", 那么会直接得到latex公式形式.

因此是否有改进的空间呢? 使得只要输出公式就能转为latex, 期待!

**Comment by binary-husky** [comment]
这个是chatgpt训练的时候肯定有一部分预料是非tex格式的，可以在systemprompt中给他提要求，比如“如果输出公式请用Tex格式”

**Comment by binary-husky** [comment]
![image](https://user-images.githubusercontent.com/96192199/229997438-b2625f36-b311-45c2-9f95-4c3052393853.png)
